packages/browser/src/plugins/ajs-destination/index.ts

```diff
@@ -283,7 +283,7 @@ export class LegacyDestination implements DestinationPlugin {
       this.integration.initialize()
     }
 
-    return this.onInitialize!.then(() => {
+    return this.initializePromise.promise.then(() => {
```

Comment:
Why does this one in particular triggers and waits for `initialize`? It is already done in the `load()` right?

nit: Can we convert it into `await this.initializePromise.promise` instead of this syntax to keep it consistent to where we await `ready`?

Reply:
It's been like this for a long time - I didn't want to risk messing with it